docs/reference/slm/getting-started-slm.asciidoc

```diff
@@ -159,19 +161,26 @@ repository is lost while copying files.
         "max_count": 50
       }
     },
-    "last_success": { 
-      "snapshot_name": "nightly-snap-2019.04.24-tmtnyjtrsxkhbrrdcgg18a", <1>
-      "time_string": "2019-04-24T16:43:49.316Z", <2>
+    "last_success": { <1>
```

Comment:
It's sometimes helpful to place callouts in the same column so they're more visible.
Not a big deal though.